src/main/resources/META-INF/plugin.xml

```diff
@@ -102,7 +102,7 @@
 
 <h3 id="-usage-"><strong>Usage</strong></h3>
 <p>
-    After Tabnine’s installation, navigate to the Tabnine Settings page (‘Tabnine: Open Settings’ from the <a href="https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette">Command Palette</a>) and verify that Tabnine local model is successfully loaded (as shown in the following screenshots)
+    After Tabnine’s installation, navigate to the Tabnine's plugin section (go to -> file -> settings -> plugins) and verify that Tabnine local model is successfully loaded.
```

Comment:
Replace the `'` with HTML compatible symobl, such as `&apos;`. See: https://www.w3.org/wiki/Common_HTML_entities_used_for_typography